README.md

```diff
@@ -1,2 +1,4 @@
 # APMAE4302homework0
 For APMAE4302 homwork0 question 2 PR test
+
+The Answer to the Great Question... Of Life, the Universe and Everything... Is... Forty-two,' said Deep Thought, with infinite majesty and calm.
```

Comment:
Good change. Maybe we need to start with a " ' " to make this sentence complete.